pyconfig.h.in

```diff
@@ -571,6 +571,9 @@
 /* Define to 1 if you have the <libutil.h> header file. */
 #undef HAVE_LIBUTIL_H
 
+/* Define to 1 if you have the `uuid' library (-luuid). */
```

Comment:
It's not the uuid library, it's specifically  uuid_generate_time_safe() (which exists on Linux, but not on OS X, even though OS X *does* have libuuid).

Reply:
That looks autogenerated. The configure check is strange, but works here on Linux:

With the patch:

   `checking for uuid_generate_time_safe in -luuid... yes`

Now use a non-existing function name:

  `AC_CHECK_LIB(uuid, uuid_generate_time_unsafe)`

  `checking for uuid_generate_time_unsafe in -luuid... no`


I did not look at the autogenerated `configure`, but it seems to work here.